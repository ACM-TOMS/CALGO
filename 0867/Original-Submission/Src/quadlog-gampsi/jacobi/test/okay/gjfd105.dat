### Numerical integration with gjqfd()
###
### int((1+x)^p * (1-x)^alpha * (1+x)^beta * ln((1+x)/2), x = -1..+1)
###
### Line format: 3 p alpha beta neval result relerr abserr
###
