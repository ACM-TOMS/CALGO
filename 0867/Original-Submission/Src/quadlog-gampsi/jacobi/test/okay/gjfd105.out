  Test integral: int((1+x)^p * (1-x)^alpha * (1+x)^beta * ln((1+x)/2),  x = -1..+1)

  1 ulp =   2.22E-16

  Floating-point precision appears to be (1 - log2(ulp) =) 53 bits

  alpha =   1.000000  beta =   1.000000     nquad =      0

  alpha =  -2.000000  beta =   1.000000     nquad =      1

  alpha =   1.000000  beta =  -2.000000     nquad =      1

  alpha =   1.000000  beta =   1.000000     nquad =  32767

  Done
